<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoPlan</class>
 <widget class="QWidget" name="AutoPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自动规划</string>
  </property>
  <widget class="QTableWidget" name="m_tableWidget_points">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_label_horizonDistance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>间距(m)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineEdit_horizonalDistance">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineEdit_verticalDistance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label_verticalDistance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>疏密(m)：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>77</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="m_checkBox_isEnableClick">
      <property name="text">
       <string>鼠标左击</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btn_add">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btn_delete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btn_commit">
      <property name="text">
       <string>预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btn_ok">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="m_label_angle">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>航向(N):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineEdit_angle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
